spec/maestro_spec.rb

```diff
@@ -36,16 +44,49 @@ def json_response
     end
   end
 
+  describe "Valid command" do
+    context "command with no args" do
+      let(:text) { "stop" }
+      let(:expected_command) { :stop }
+
+      it_behaves_like "a valid maestro command"
+    end
+
+    context "command with args" do
+      let(:text) { "pos #{time}" }
+      let(:expected_command) { :pos }
+
+      context "with a valid number" do
+        let(:time) { "34" }
+
+        it_behaves_like "a valid maestro command" do
+          let(:expected_args) { time }
+        end
+      end
+    end
+  end
+
   describe "Invalid command" do
     let(:text) { "bogus" }
 
     it "returns the usage text" do
-      expect(last_response).to_not be_ok
+      subject
+      expect(last_response).to be_ok
       expect(json_response["response_type"]).to eq("in_channel")
       expect(json_response["text"]).to include("Usage:")
       expect(json_response["text"]).to include("/maestro")
       expect(json_response["text"]).to_not include("CLIENT_ID")
       expect(json_response["text"]).to_not include("CLIENT_SECRET")
     end
   end
+
+  describe "Bash command execution" do
+    let(:text) { "status; ls" }
+    it "returns the usage text" do
+      subject
+      expect(last_response).to be_ok
```

Comment:
Since we've got this set of `expect`s a few times now, we should probably move it to a shared_examples block